tests/RepositoryForms/FieldType/FieldTypeFormMapperDispatcherTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryForms\Tests\FieldType;
+
+use eZ\Publish\API\Repository\Values\Content\Field;
+use eZ\Publish\Core\Repository\Values\ContentType\FieldDefinition;
+use EzSystems\RepositoryForms\Data\Content\FieldData;
+use EzSystems\RepositoryForms\Data\ContentTypeData;
+use EzSystems\RepositoryForms\Data\FieldDefinitionData;
+use EzSystems\RepositoryForms\FieldType\FieldTypeFormMapperDispatcher;
+use EzSystems\RepositoryForms\FieldType\FieldTypeFormMapperDispatcherInterface;
+use PHPUnit_Framework_TestCase;
+
+class FieldTypeFormMapperDispatcherTest extends PHPUnit_Framework_TestCase
+{
+    /**
+     * @var FieldTypeFormMapperDispatcherInterface
+     */
+    private $dispatcher;
+
+    /**
+     * @var \EzSystems\RepositoryForms\FieldType\FieldDefinitionFormMapperInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $fieldDefinitionMapperMock;
+
+    /**
+     * @var \EzSystems\RepositoryForms\FieldType\FieldValueFormMapperInterface|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $fieldValueMapperMock;
+
+    public function setUp()
+    {
+        $this->dispatcher = new FieldTypeFormMapperDispatcher();
+
+        // Should be improved to test with a value + definition mapper (this is what repo-forms uses)
```

Comment:
Well, I'd rather go for a `@todo` in the code, but the symfony guidelines don't like those...